src/pages/news/SingleNews.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Button } from '@/components/Button'
+import ChangeColor from '@/components/IconSVG/ChangeColor'
+
+const SingleNews = ({
+  title,
+  description,
+  imageUrl,
+  index,
+}: {
+  title: string
+  description: string
+  imageUrl: string
+  index: number
+}) => {
+  return (
+    <div className="container">
+      <div className="my-10 grid grid-cols-2 items-center rounded-lg border-2 border-solid border-gray-thin p-8 hover:border-primary">
+        <div className={`${index % 2 === 1 ? 'order-last' : ''} mr-4 space-y-4`}>
```

Comment:
Let's keep this in variable to not declare each time

Reply:
Done